feat: add checklist to pull request template (#10286)
terrapkg/packages: .github/PULL_REQUEST_TEMPLATE/new_package.md

**Additional context**
Add any other context about the package submission here. Link to any relavent issues.

**Checklist**

> - [] This package is maintained OR there is a valid reason to add it (e.g. python dependency)

Reviewer: Python is ~slop~ a proper noun.

> - [] This package is maintained OR there is a valid reason tk add it if not (ex. python dependancy)

Reviewer 2: ```suggestion
- [] This package is maintained OR there is a valid reason to add it (e.g. python dependency)
```

**Additional context**
Add any other context about the package submission here. Link to any relavent issues.

**Checklist**
- [] I have tested at least the `x86_64` version of the package
- [] I have read through any relevant [Terra](https://developer.fyralabs.com/terra) and [Fedora packaging](https://docs.fedoraproject.org/en-US/packaging-guidelines/) documentation/policies/guidelines
- [] I have made sure there are no security issues with this package to the best of my ability

> - [] I have made sure this is not in Fedora (unless adding to the [extras repo](https://developer.fyralabs.com/terra/installing#extras)).

Reviewer 3: they should provide a reason why we need it 

also first check box on the page "this software is actively maintained"

Reviewer: The actively maintained part falls apart with Python dependencies and it makes me wish for death.
